### PR TITLE
XSUP-29866

### PR DESCRIPTION
**Pull request by kobymeir**
## Status
- [x] Ready

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-29866

## Description
- Fixed an issue with rendering Markdown in the email body to HTML.
- Fixed an issue with the `body_type` argument missing from the documentation.
- Updated the Docker image to: *demisto/bs4-py3:1.0.0.86348*.

### CODE REVIEW

File: Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.py [hunk 1]
```diff
@@ -616,13 +631,15 @@ def format_body(new_email_body):
...
-    html_body = res[0]['Contents']
-
-    return html_body
+    return markdown(new_email_body,
+                    extensions=[
+                        'tables',
+                        'fenced_code',
+                        'legacy_em',
+                        'sane_lists',
+                        'nl2br',
```
Comment (kobymeir) [inline Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.py:640]: the nl2br replaces the old string replacements

File: Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.py [hunk 2]
```diff
@@ -970,8 +987,7 @@ def main():
     email_selected_thread = custom_fields.get('emailselectedthread')
     subject_include_incident_id = argToBoolean(args.get('subject_include_incident_id', False))
     body_type = args.get('bodyType') or args.get('body_type') or 'html'
-
-    argToBoolean(args.get('reputation_calc_async', False))
+    reputation_calc_async = argToBoolean(args.get('reputation_calc_async', False))
```
Comment (kobymeir) [inline Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.py:990]: old bug fix, not related to the current issue

**Review by DeanArbel — APPROVED** [review]
Very nice work.
What about unit tests?

**Comment by kobymeir** [comment]
@DeanArbel added unit tests

**Pull request merged**